src/xyz/derkades/serverselectorx/ItemMoveDropCancelListener.java

```diff
@@ -32,6 +33,7 @@ public void onDrop(final PlayerDropItemEvent event){
 			Bukkit.getPluginManager().registerEvents(new Listener() {
 				@EventHandler(ignoreCancelled = true, priority = EventPriority.LOW)
 				public void onItemMove(final InventoryClickEvent event){
+					if (inventory.getBoolean("cancel-item-move-player-only") && !event.getClickedInventory().getType().equals(InventoryType.PLAYER)) return;
```

Comment:
getClickedInventory() is nullable, might want to check that

Reply:
is the most recent what you were looking for?